collectors/hra/parser.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .record import Record
+from datetime import date
+import logging
+import six
+logger = logging.getLogger(__name__)
+
+
+def parse_response(response, endpoint, from_date, to_date, errors, success, conn):
+    for application in response.json():
+        try:
+            data = {}
+            data['last_updated'] = date.today().strftime('%Y-%m-%d')
+            data['api_date_from'] = from_date.strftime('%Y-%m-%d')
+            data['api_date_to'] = to_date.strftime('%Y-%m-%d')
```

Comment:
> About parse_response - correct me if I'm wrong but it looks like code just returns on first iteration of for-in cycle instead of yielding all records (if response.json() has many applications).

@roll True, I haven't been very careful here.